example/cycle/test/test.js

```diff
@@ -7,31 +7,25 @@ import * as actions from '../../actions';
 
 import { fetchReposByUser, searchUsers } from '../';
 
-function assertSourcesSinks(sources, sinks, main, done) {
-  const Time = mockTimeSource();
+function assertSourcesSinks(sources, sinks, main, done, timeOpts = {}) {
```

Comment:
@Widdershin the DOM driver is a bit tricker but perhaps something like this? https://gist.github.com/lmatteis/b1d18f863f589c496dc604b95ca56454

Since I simply pass the `domSink` object to `Time.diagram` I'm not sure it's capable of deep asserting a vtree.